src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -3166,8 +3170,8 @@ class CgroupUtils(object):
             avail_mem = available['memory']
             if req_mem > avail_mem:
                 pbs.logmsg(pbs.EVENT_DEBUG4,
-                           "Insufficient memory on socket(s) " +
-                           "%s: requested:%s, assigned:%s" %
+                           'Insufficient memory on socket(s) ' +
```

Comment:
I'd replace this + with a '\\'.  This way the string doesn't need to be regenerated.

Reply:
Placed both strings within parens and removed the + operator.